### ISSUE TITLE
tsqlanalyze - sql errors not reported in output file

### ISSUE THREAD

**Issue by beorn107** [ErikEJ/SqlServer.Rules#331]
Hi when using tsqlanalyze I find that errors are not included in the output file.

Example table script:
CREATE TABLE [dbo].[foo] (
    [fooId] BIGINT  NOT NULL,
    [fooType]        TINYINT NOT NULL
	[columnblah]		nvarchar(50)
)
GO

ALTER TABLE [dbo].[foo] ADD CONSTRAINT [PK_foo]
    PRIMARY KEY CLUSTERED ([fooId] ASC)
        WITH (ONLINE = ON)
        ON [PRIMARY]
GO
 
Saved as foo.sql in c:\Temp

executing the following command: 

tsqlanalyze -i "C:\Temp\foo.sql"

returns the following:

T-SQL Analyzer CLI 1.0.34
https://github.com/ErikEJ/SqlServer.Rules

error: C:\Temp\foo.sql(1,1): Error SQL46010: Incorrect syntax near '[columnblah]'.

Analyzed 1 files in 1.864 seconds using 'Microsoft.Rules.Data, Smells, SqlServer.Rules'. 0 problems found.

changing the command to:

tsqlanalyze -i "C:\Temp\foo.sql" -o C:\Temp\result.xml

Result.xml:

```xml
<?xml version="1.0" encoding="utf-8"?>
<Problems />
```

If analyzing several files in a directory some which have warning generated and others have errors only the warnings are exported to the output file.  Is that intentional?  Could errors be included in the output file?

**Comment by ErikEJ** [comment]
@beorn107 Fixed initially by creating a buddy .txt file with the model errors.
